app/views/claims/bank_details.html.erb

```diff
@@ -19,22 +20,22 @@
             <%= form.label :bank_sort_code, "Sort code", class: "govuk-label" %>
             <span  id="sort-code-hint" class="govuk-hint">For example: 44 00 26</span>
             <%= form.text_field :bank_sort_code,
-              class: "govuk-input govuk-!-width-one-quarter",
+              class: css_classes_for_input(current_claim, :bank_sort_code, "govuk-!-width-one-quarter"),
               "aria-describedby" => "sort-code-hint"
             %>
           </div>
 
           <div class="govuk-form-group">
             <%= form.label :bank_account_number, "Account number", class: "govuk-label" %>
             <span  id="account-number-hint" class="govuk-hint">For example: 70 87 24 90</span>
-            <%= form.text_field :bank_account_number,
-              class: "govuk-input govuk-input--width-20",
+            <%= form.text_field :bank_account_number,  class: css_classes_for_input(current_claim, :bank_account_number, "
```

Comment:
As above, do we also need a `<%= errors_tag current_claim, :bank_account_number %>` to go with this text field?

Reply:
oh…